diff --git a/node_modules/expo-router/ios/LinkPreview/LinkPreviewNativeActionView.swift b/node_modules/expo-router/ios/LinkPreview/LinkPreviewNativeActionView.swift
index 3884b78..b3901f0 100644
--- a/node_modules/expo-router/ios/LinkPreview/LinkPreviewNativeActionView.swift
+++ b/node_modules/expo-router/ios/LinkPreview/LinkPreviewNativeActionView.swift
@@ -137,7 +137,7 @@ class LinkPreviewNativeActionView: RouterViewWithLogger, LinkPreviewMenuUpdatabl
     baseUiAction.attributes = attributes
     baseUiAction.state = isOn == true ? .on : .off
 
-    if let subtitle = subtitle {
+    if #available(iOS 16.0, *), let subtitle = subtitle {
       baseUiAction.subtitle = subtitle
     }
     if let label = discoverabilityLabel {
